RedHatInsights/insights-inventory-frontend
# fix(RHINENG-9687): Push system type filter into URL, apply on refresh

Fixes https://issues.redhat.com/browse/RHINENG-9687.

This PR is intended to address the comments related to the system type filter not being pushed into the URL and also not being reapplied between Conventional and Immutable tabs. 

To test:
1. Run the PR
2. Apply one/both of the system filter options
3. Observe that now the filter is applied to URL and you can open the page with the same filter in a new browser tab
4. Apply the filter, and switch to the immutable tab. 
5. Observe that when you switch back to conventional tab, still the previous system filter is still present
